src/flags/parse.test.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+
+import Base from '../command'
+import NumberFlag from './string'
+
+let flagParser = function (n: number) : Promise<> {
```

Comment:
this doesn't work with flow since it changes the type from string which makes it unhappy. I wasn't able to find a way around this so I think the best way to do this (unless we can figure out a better way) is to do something like we do for NumberFlag where we just duplicate all the code from StringFlag